src/orgmode/inline.clj

```diff
@@ -147,3 +151,12 @@
       (make-elem (fmt-re "_")   (fmt-create :underline))
       (make-elem (fmt-re "=")   (fmt-create :verbatim))
       (make-elem (fmt-re "~")   (fmt-create :code))))
+
+
+(comment
```

Comment:
I think that we can remove this comment and turn it into a test?

Reply:
Okay.